CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # 1.0.10
 
--adds NPM keywords to help people actually finds the package
+-adds NPM keywords to help people actually find the package
 -renames removeLocationStream to removeLocation. the former name was an artifact of an older version of this package and since it doesn't stream anything anymore, i figured to take it out to avoid confusion.
```

Comment:
`I` wasn't capitalized, but I think there is a bunch of formatting changes to make this a bit more professional. 
```suggestion
- Renames `removeLocationStream` to `removeLocation`. The former name was an artifact of an older version of this package, and since it doesn't stream anything anymore, I figured to take it out to avoid confusion.
```